### PR TITLE
Adds Telco bug for 4.10.30 release

### PR DESCRIPTION
**Pull request by opayne1**
This PR adds a Telco bug for the 4.10.30 release. It was my interpretation that this was going in for 4.10.31 due to the bump of 4.10.29 to 4.10.30. Upon discussing with @EinatGlottmann it was this 4.10.30 the note needed to be in. cc @kquinn1204 

Link to docs preview:
http://file.rdu.redhat.com/opayne/add-bond-cni-4.10.30/release_notes/ocp-4-10-release-notes.html#ocp-4-10-30

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3357,6 +3357,18 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.10.30 --pullspecs
 ----
 
+[id="ocp-4-10-30-bug-fixes"]
+==== Bug fixes
+
+* Previously, the functionality of Bond-CNI was limited to only active-backup mode. With this update, the bonding modes supported are:
+
+** `balance-rr - 0`
```
Comment (kelbrown20) [inline release_notes/ocp-4-10-release-notes.adoc:3365]: Question. In this module, the modes are formatted like this:
`balance-rr` - 0
`active-backup` - 1
`balance-xor` - 2

Module: https://github.com/openshift/openshift-docs/blob/main/modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc
Docs: https://docs.openshift.com/container-platform/4.11/networking/hardware_networks/using-pod-level-bonding.html#nw-sriov-cfg-creating-bond-network-attachment-definition_using-pod-level-bonding

So I wasn't sure which one was right

**Review by kelbrown20 — COMMENTED** [review]
Hi Olivia! Just a small question, otherwise this LGTM!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:3365]: ooo thanks for the comment. Need to fix 4.11 note then too

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc]: updated!

**Comment by mmirecki** [comment]
LGTM

**Comment by EinatGlottmann** [comment]
LGTM

**Pull request merged**